### ISSUE TITLE
fnf mobile why sometimes crashes reason why

### ISSUE THREAD

**Issue by King-Fox-Yt** [luckydog7/Funkin-android#594]
there's a bug when the song end it crash u
btw I have a mid range phone or high end range

**Comment by DanisLixo** [comment]
why didn't you use BUG REPORT?

**Comment by DanisLixo** [comment]
why?

**Comment by DabKnight** [comment]
It cant be fixed.
It just happens.

**Comment by DabKnight** [comment]
Week 5 at the latest version just crashes for me.
There's nothing i can do about it.

**Comment by GhryleFunkin** [comment]
> It cant be fixed.
> It just happens.

Yes we need higher ram for it because it will crash if the ram is under 4gb

**Comment by King-Fox-Yt** [comment]
> > It cant be fixed.
> > It just happens.
> 
> Yes we need higher ram for it because it will crash if the ram is under 4gb

no u idiot it's a random crash

**Comment by DanisLixo** [comment]
> > > It cant be fixed.
> > > It just happens.
> > 
> > 
> > Yes we need higher ram for it because it will crash if the ram is under 4gb
> 
> no u idiot it's a random crash

no u r much more idiot, if you reported the bug, you were able to see #12 and #10

**Comment by GhryleFunkin** [comment]
> > > It cant be fixed.
> > > It just happens.
> > 
> > 
> > Yes we need higher ram for it because it will crash if the ram is under 4gb
> 
> no u idiot it's a random crash

Bruh ok I'm an idiot

**Comment by GhryleFunkin** [comment]
> > > It cant be fixed.
> > > It just happens.
> > 
> > 
> > Yes we need higher ram for it because it will crash if the ram is under 4gb
> 
> no u idiot it's a random crash



> > > > It cant be fixed.
> > > > It just happens.
> > > 
> > > 
> > > Yes we need higher ram for it because it will crash if the ram is under 4gb
> > 
> > 
> > no u idiot it's a random crash
> 
> Bruh ok I'm an idiot

Anyways you were dumbass lol because 1 song is taking you're 2gb ram so it's crashed when you play the game at 2 gb now you understand crap?
